docs/guides/stay-up-to-date/terraform/how-to-update-to-aws-provider-v4/deployment-walkthrough.md

```diff
@@ -0,0 +1,148 @@
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-vpc/releases/tag/v0.22.0">v0.22.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-asg</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-asg/releases/tag/v0.19.0">v0.19.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-server</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-server/releases/tag/v0.15.0">v0.15.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-lambda</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-lambda/releases/tag/v0.20.0">v0.20.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-security</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-security/releases/tag/v0.65.0">v0.65.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-load-balancer</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-load-balancer/releases/tag/v0.29.0">v0.29.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-data-storage</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-data-storage/releases/tag/v0.24.0">v0.24.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-cache</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-cache/releases/tag/v0.18.0">v0.18.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-messaging</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-messaging/releases/tag/v0.9.0">v0.9.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-static-assets</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-static-assets/releases/tag/v0.15.0">v0.15.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-monitoring</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-monitoring/releases/tag/v0.34.1">v0.34.1</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-openvpn</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-openvpn/releases/tag/v0.24.0">v0.24.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-ecs</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-ecs/releases/tag/v0.34.0">v0.34.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-ci</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-ci/releases/tag/v0.48.0">v0.48.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-eks</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-eks/releases/tag/v0.53.0">v0.53.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-service-catalog</p></td>
```

Comment:
Do any / some / a lot of our customers only consume from the service catalog instead of going down lower into individual modules? I wonder whether the service catalog / CIS service catalog should be called out separately.

Reply:
These are listed in order of dependency, so the ones at the top should be updated first before the ones toward the bottom. That was the idea. I'm not sure if the service catalogs should be in a separate table. 🤔

Reply:
I know that users of the Reference Architecture will want to update some of the individual modules too, because the service catalog doesn't cover everything in the Ref Arch. Hmm.